test/glob.js

```diff
@@ -91,11 +91,12 @@ test('glob: options.cwd', async t => {
   let dir = join(cwd, 'one', 'child');
 
   await isMatch(t, '../*', { cwd:dir }, [
+    '',
     '../a.js',
     '../a.md',
     '../a.txt',
     '../b.txt',
-    '../child'
+    // '../child' is in fact resolved to '' related to himself !
```

Comment:
I think this was working as expected. Pretty sure that's how other glob implementations works, including `bash` and other JS packages like `node-glob`. Instead of returning an empty string it should return the name of the folder

Reply:
I believe so too